Tests/UIKitTests/UILayoutPriorityExtensionsTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UILayoutPriorityExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by diamantidis on 8/19/18.
+//  Copyright © 2018 SwifterSwift. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+import XCTest
+@testable import SwifterSwift
+
+final class UILayoutPriorityExtensionsTests: XCTestCase {
+
+    // MARK: - Initializers
+    func testFloatLiteralInitializer() {
+        let constraint = NSLayoutConstraint(
```

Comment:
Why are you creating this type of object? Why not just set
```Swift
var priority: UILayoutPriority = 0.5
```
?
I used a `var` so that you can test changing the value as well.

This comment applies for the other test too.

Reply:
I have updated the unit tests and I change the assertions to compare against `UILayoutPriority(rawValue: priority)`, since it should have the same behavior as this call.

For the `Float.greatestFiniteMagnitude` and `Int.max`, I had to add the literal value